curvefs/src/tools/curvefs_tool_define.h

```diff
@@ -26,48 +26,95 @@
 #include <gflags/gflags.h>
 
 #include <functional>
+#include <sstream>
 #include <string>
+#include <vector>
 
+#include "curvefs/proto/copyset.pb.h"
+#include "curvefs/proto/heartbeat.pb.h"
 #include "src/common/configuration.h"
 
 namespace curvefs {
 namespace tools {
 
+/* command */
 // programe name
 const char kProgrameName[] = "curvefs-tool";
-
 // version
 const char kVersionCmd[] = "version";
-
 // build-topology
 const char kBuildTopologyCmd[] = "build-topology";
-
 // umount
-const char kUmountCmd[] = "umountfs";
+const char kUmountCmd[] = "umount-fs";
+// metadata-usage
+const char kMetedataUsageCmd[] = "usage-metadata";
+// mds-status
+const char kMdsStatusCmd[] = "status-mds";
+// metaserver-status
+const char kMetaserverStatusCmd[] = "status-metaserver";
+// etcd-status
+const char kEtcdStatusCmd[] = "status-etcd";
+// copysets-status
+// check the cluster of copysets status
+const char kCopysetsStatusCmd[] = "status-copysets";
+// copyset-query
+const char kCopysetQueryCmd[] = "query-copyset";
+// list-fs-partition
+const char kFsPartitionListCmd[] = "list-fs-partition";
+// fsinfo-list
+const char kFsInfoListCmd[] = "list-fs";
+// list-fs-copysetid
+const char kFsCopysetIdListCmd[] = "list-fs-copysetid";
+// no-invoke Used for commands that are not directly invoked
+const char kNoInvokeCmd[] = "no-invoke";
 
 // configure
 const char kConfPathHelp[] = "[-confPath=/etc/curvefs/tools.conf]";
-
 // kHelp
 const char kHelpStr[] =
-    "Usage: curve_ops_tool [Command] [OPTIONS...]\n"
+    "Usage: curvefs_tool [Command] [OPTIONS...]\n"
     "COMMANDS:\n"  // NOLINT
     "version: show the version of cluster\n"
     "build-topology: build cluster topology based on topo.json\n"
-    "umountfs: umount curvefs from local and cluster"
+    "umount-fs: umount curvefs from local and cluster\n"
+    "usage-metadata: show the metadata usage of cluster\n"
+    "status-mds: show the status of mds\n"
+    "status-metaserver-: show the status of metaserver\n"
```

Comment:
status-metaserver- not status-metaserver ?

Reply:
> status-metaserver- not status-metaserver ?

fix